lib/shitty_linq_ex.ex

```diff
@@ -318,6 +320,46 @@ defmodule ShittyLinqEx do
     take_list(source, count)
   end
 
+  @doc """
+  Returns a new enumerable collection that contains the last count elements from source.
+
+  ## Parameters
+
+  - `source`: A sequence of values to take.
+  - `count`: The number of elements to take from the end of the collection.
+
+  ## Returns
+
+  A new enumerable collection that contains the last count elements from source.
+
+  ## Examples
+
+    iex> import ShittyLinqEx, only: [take_last: 2]
+    iex> take_last(["A", "B", "C"], 2)
+    ["C", "B"]
```

Comment:
`take_last` should keep the items in their original order. You'll probably need to `reverse` the list one more time before returning.

Reply:
oh man! Sorry, I didn't notice it in the documentation, I'll correct it ;)